database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -11,16 +11,20 @@ class CreateUsersTable extends Migration
      *
      * @return void
      */
+    protected $fillable = ['id', 'name','email',];
+
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
+            /*
```

Comment:
Don't worry about commenting this at, we will use the `User` model at some point in the future.